plaidml/core/core.h

```diff
@@ -10,9 +10,12 @@
 #include <vector>
 
 #include "plaidml/core/ffi.h"
+#include "pmlc/util/enums.h"
```

Comment:
We need to keep the `pmlc` and `plaidml` APIs distinct. The idea is that end-users will pull in `plaidml/edsl/edsl.h`, and when they do they should only be including `plaidml` because `pmlc` might be in a different `.so`.